src/cljs/rems/dropdown.cljs

```diff
@@ -45,12 +39,18 @@
      (component-info dropdown)
      (example "dropdown menu, single-choice, empty"
               [dropdown {:items items
-                         :item-key :userid
+                         :item-key :id
                          :item-label :userid
                          :on-change on-change}])
+     (example "dropdown menu, single-choice, selected item Bob"
+              [dropdown {:items items
+                         :item-key :id
+                         :item-label :userid
+                         :item-selected? #(= "Bob" (:userid %))
```

Comment:
Renaming `:userid` to `:name` would make the example more understandable, when there is only one unique ID. Also, `:item-selected?` should check the `:id`.

Reply:
Yes I think I'll do that. Only for us `:userid` is a string obviously.

I decided to use `"Bob"` here so that you don't have to look for the id value mapping from source in the guide.

Reply:
Also it demonstrates that you get the full object. The last example uses the `:id` instead.

Reply:
👌